consensus/istanbul/backend/announce.go

```diff
@@ -419,3 +516,125 @@ func (sb *Backend) regossipIstAnnounce(msg *istanbul.Message, payload []byte, an
 
 	return nil
 }
+
+// ===============================================================
+//
+// define the AnnounceVersion messge format, the getAnnounceVersions send and handle function, and the announceVersions handle function
+
+type announceVersion struct {
+	ValAddress           common.Address
+	AnnounceMsgTimestamp uint
+}
+
+// EncodeRLP serializes av into the Ethereum RLP format.
```

Comment:
Nit: Took me a few seconds to figure out what `av` was.
```suggestion
// EncodeRLP serializes announceVersion into the Ethereum RLP format.
```